### ISSUE TITLE
Zinc incorrectly labeled as a transition metal.

### ISSUE THREAD

**Issue by micmou** [AllTheMods/ATM-8#369]
The title pretty much states the issue I have but I also made a sexy meme for your viewing pleasure.
https://i.imgur.com/Icfod6h.jpeg

Zinc sits in the Group 12 section of elements notated as IIB

**Comment by Timbroglio** [comment]
Hi! I'm one of the developers for Chemlib, the mod that adds chemical elements to the game.
Zinc, cadmium, and mercury are generally considered controversial for their inclusion in transition metals due to their properties not perfectly fitting into the category, and I understand and recognize the IUPAC standards do not recognize them as transition metals.

However, this mod is meant to be accessible to all players, including those with no experience or only a surface-level understanding of chemistry. As such, I've made multiple decisions in regards to the specifics in order to simplify things beyond IUPAC standards. There are degrees of inconsistency among the categorization of group 2B, polonium, oganesson, flerovium,  and others. Discussing and explaining these inconsistencies is outside the scope of this mod.

TL;DR
I made this periodic table, I do what I want 🕶️ 
Also, Education Edition puts zinc as a transition metal. Maybe put in a bug report with them first? If they change, I'll consider updating it.

**Comment by Mitchell5200** [comment]
Closing as mod dev says "Working as intended".
